include/dxc/Support/HLSLOptions.td

```diff
@@ -311,6 +311,8 @@ def print_before_all : Flag<["-", "/"], "print-before-all">, Group<hlslcomp_Grou
   HelpText<"Print LLVM IR before each pass.">;
 def print_before : Separate<["-", "/"], "print-before">, Group<hlslcomp_Group>, Flags<[CoreOption, HelpHidden]>,
   HelpText<"Print LLVM IR before a specific pass. May be specificied multiple times.">;
+def select_validator : Separate<["-", "/"], "select-validator">, Group<hlslcomp_Group>, Flags<[CoreOption, HelpHidden]>,
```

Comment:
Wasn’t this change supposed to be broken out from the SM 6.8 changes and merged separately?

Reply:
It was, but we couldn't test it without a test lib. That change is still blocked from going into main and I would say we determined it wasn't necessary, but more accurately I said I didn't think it was and no one disagreed. Do you disagree?

Reply:
Issue created to track this: https://github.com/microsoft/DirectXShaderCompiler/issues/5991